docs/api/go/getting-started.md

```diff
@@ -0,0 +1,209 @@
+# Getting started
+
+In this guide we will demonstrate how to:
+
+1. Connect to the zkSync network.
+2. Deposit assets from Ethereum into zkSync.
+3. Check Balance 
+4. Transfer funds (ERC20 tokens) into the zksync network
+5. Withdraw funds (Native coins)
+
+## Prerequisite
+
+This guide assumes that you are familiar with the [Go programming](https://go.dev/doc/) language.
+Also, the required version is >= 1.17 and Go modules are required.
+
+ 
+## Installation
+
+To install the SDK with the `go get` command, run the following:
+
+```go 
+
+get github.com/zksync-sdk/zksync2-go
+
+```
+## Instantiating the SDK
+
+To start using this SDK, you just need to pass in a provider configuration.
+
+Once you have all the necessary dependencies, you can follow the setup steps to get started using the SDK.
+Using these three instances `ZkSync Provider`, `EthereumProvider` and `Wallet`, you can perform all basic actions with ZkSync network.
+
+```go
+
+package main
+
+import (
+    "github.com/ethereum/go-ethereum/rpc"
+    "github.com/zksync-sdk/zksync2-go"
+)
+
+// first, init Ethereum Signer, from your mnemonic
+es, err := zksync2.NewEthSignerFromMnemonic("<mnemonic words>", zksync2.ZkSyncChainIdMainnet)
```

Comment:
in general, I think we should improve variable names in all code samples. `es` stands for ethereum signer, so let's name it that way. Same with `zp` (zksync provider) or `w` (wallet)